src/lib/curl/base.h

```diff
@@ -33,6 +33,20 @@ extern "C" {
 #include <freeradius-devel/server/request.h>
 #include <freeradius-devel/util/event.h>
 
+#include <freeradius-devel/server/base.h>
```

Comment:
It shouldn't include every other header in the server library, especially as it's not linked against it